comicScraper.py

```diff
@@ -154,9 +166,12 @@ def main(fullComicDownload, singleIssueDownload, title):
     imageLinks = []
     for issueLink in issueLinks:
         issueName = getIssueName(issueLink, startURL)
+        print(f"Pulling images for issue {issueName}")
         issueImageLinks = scrapeImageLinksFromIssue(issueLink)
-        # TODO: add a check here for /special/areyouhuman
-        # TODO: kick into a chrome window and wait for user input
+        print(f"\nissueLink is {issueLink}")
+        print(f"images are: {issueImageLinks}\n")
+        if dealWithCaptcha(issueImageLinks):
```

Comment:
The issue links seem to work fine even when we've been captcha snapped, it's the image links from within the issues that fail. This check could go under the "scrapeImageLinksFromIssue" function
`def scrapeImageLinksFromIssue(url):
    req = requests.get(url, headers)
    soup = bs(req.content, 'html.parser')
    soup = soup.prettify()
    lines = soup.split("\n")
    imageLinks = []

    for line in lines:
        if "https://2.bp.blogspot.com" in line:
            imageUrl = extractImageUrlFromText(line)
            imageLinks.append(imageUrl)

        if "AreYouHuman" in line:
            print("Captcha Detected, Opening Browser")
            webbrowser.open(url)
            input("\nPlease complete the captcha and press enter\n")
            return scrapeImageLinksFromIssue(url)

    return imageLinks`

Reply:
as of right now the captcha checker is being called on `issueImageLinks` between `scrapeImageLinksFromIssue` calls. Are you thinking we should add the captcha checker exclusively to `scrapeImageLinksFromIssue` and handle captcha issues there?

Reply:
I've removed all the checks for now till we figure out the best place to add it to. the function will stay